### ISSUE TITLE
Log verbosity level

### ISSUE THREAD

**Issue by pshriwise** [gprt-org/GPRT#29]
There are still a few direct calls to `std::cout` hanging around. I think a setting for the verbosity of log output would be useful.

**Comment by natevm** [comment]
for example, GeomType currently uses a std::cout in the constructor / destructor. That could instead be piped through some function printing more / less depending on verbosity.

I'm wondering if it makes sense to do anything other than "verbose / not verbose". like, low vs medium vs high. 

Could also setup warnings, error, and info or something like that.

**Comment by pshriwise** [comment]
Yeah I think having different classes of messaging is another solution. I'm good w/ that.
